src/chordsymbol.ts

```diff
@@ -280,108 +148,67 @@ export class ChordSymbol extends Modifier {
     if (!symbols || symbols.length === 0) return false;
 
     let width = 0;
-    let nonSuperWidth = 0;
     let leftWidth = 0;
     let rightWidth = 0;
     let maxLeftGlyphWidth = 0;
     let maxRightGlyphWidth = 0;
 
     for (const symbol of symbols) {
-      const fontSize = Font.convertSizeToPointValue(symbol.textFont?.size);
-      const fontAdj = Font.scaleSize(fontSize, 0.05);
-      const glyphAdj = fontAdj * 2;
       const note: Note = symbol.checkAttachedNote();
-      let symbolWidth = 0;
       let lineSpaces = 1;
-      let vAlign = false;
 
       for (let j = 0; j < symbol.symbolBlocks.length; ++j) {
         const block = symbol.symbolBlocks[j];
-        const sup = ChordSymbol.isSuperscript(block);
-        const sub = ChordSymbol.isSubscript(block);
-        const superSubScale = sup || sub ? ChordSymbol.superSubRatio : 1;
-        const adj = block.symbolType === SymbolTypes.GLYPH ? glyphAdj * superSubScale : fontAdj * superSubScale;
+        const sup = block.isSuperscript();
+        const sub = block.isSubscript();
+        block.setXShift(width);
 
         // If there are super/subscripts, they extend beyond the line so
         // assume they take up 2 lines
         if (sup || sub) {
           lineSpaces = 2;
         }
 
-        // If there is a symbol-specific offset, add it but consider font
-        // size since font and glyphs will be interspersed.
-        const fontSize = symbol.textFormatter.fontSizeInPixels;
-        const superSubFontSize = fontSize * superSubScale;
-        if (block.symbolType === SymbolTypes.GLYPH && block.glyph !== undefined) {
-          block.width = ChordSymbol.getWidthForGlyph(block.glyph) * superSubFontSize;
-          block.yShift += ChordSymbol.getYShiftForGlyph(block.glyph) * superSubFontSize;
-          block.xShift += ChordSymbol.getXShiftForGlyph(block.glyph) * superSubFontSize;
-          block.glyph.scale = block.glyph.scale * adj;
-        } else if (block.symbolType === SymbolTypes.TEXT) {
-          block.width = block.width * superSubFontSize;
-          block.yShift += symbol.getYOffsetForText(block.text) * adj;
-        }
-
-        if (
-          block.symbolType === SymbolTypes.GLYPH &&
-          block.glyph !== undefined &&
-          block.glyph.code === ChordSymbol.glyphs.over.code
-        ) {
-          lineSpaces = 2;
-        }
-        block.width += ChordSymbol.spacingBetweenBlocks * fontSize * superSubScale;
-
         // If a subscript immediately  follows a superscript block, try to
```

Comment:
This line has an extra space in `"immediately  follows"`. I know it might not be your comment, but we might as well fix it since we're here....

Reply:
done